packages/web/components/CardResultsProvider.tsx

```diff
@@ -1,24 +1,29 @@
 "use client";
 
 import { ICard } from "@/lib/api";
+import { supabase } from "@/lib/supabase/supabaseClient";
 import { createContext, useContext, useEffect, useState } from "react";
 
 type ResultsContext = {
   addMyCard: (card: ICard) => void;
+  fetchMoreCards: () => Promise<void>;
   cards: ICard[];
   indexedCards: Map<string, ICard>;
+  hasMoreCards: boolean;
 };
 
 const Context = createContext<ResultsContext>({
   addMyCard: () => {},
+  fetchMoreCards: async () => {},
   cards: [],
   indexedCards: new Map<string, ICard>(),
+  hasMoreCards: true,
 });
 
 const compareCards = (a: ICard, b: ICard) => {
-  if (!a.created_at) return 1;
-  if (!b.created_at) return -1;
-  return new Date(b.created_at!).getTime() - new Date(a.created_at!).getTime();
+  if (!a.id) return 1;
```

Comment:
Why this switch? Won't it cause cards not to be ordered chronologically?

Reply:
Forgot to switch back. Just experimenting. Thinking about the ML model that will be responsible for sorting. What classification schema did you have in mind?

Reply:
I think sorting will always be chronological. When we have user accounts, we can base the filtering on personal preferences. Until then, I think showing all results is fine.

Then we'll need to mix in the case studies. Those may show up multiple times for the same user. The algorithm could probably be as simple as "one case study per rand(4, 8) cards"